Kajabi/sage-lib
# [DSS-174] Add Google Analytics to Storybook

## Description
Adds Google Analytics tracking to the Sage Storybook website.
The tracking code is added by using a Storybook Addon as outlined [here](https://storybook.js.org/addons/@storybook/addon-google-analytics).


## Screenshots
![Screen Shot 2022-09-28 at 1 06 38 PM](https://user-images.githubusercontent.com/1175111/192879732-10483061-3125-4863-8a5d-b1964e099ac4.png)



## Testing in `sage-lib`
- Install Google Analytics Debugger ([extension](https://chrome.google.com/webstore/detail/google-analytics-debugger/jnkmfdileelhofjcijamephohjechhna?hl=en))
- Navigate to [Sage Storybook](http://localhost:4100/) 
- Open Chrome DevTools to Console
- Turn on the GA extension
- Refresh the page and verify ID matches `UA-238598709-4`


## Testing in `kajabi-products`
1. (**LOW**) Adds GA tracking to Sage Storybook website. No KP impact is expected.


## Related
https://kajabi.atlassian.net/browse/DSS-174
